.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,31 @@
+name: Build wheels
+
+on:
+  workflow_dispatch
```

Comment:
Ideally, we would set this up to work on releases as well, right? Or do you only want to use this workflow manually?

Reply:
> However, the repo currently has none of the automated release machinery that other Enthought projects have. So we're stuck with manual until that's sorted out. 

Yeah, I don't think it's too inconvenient to simply download the artifacts and upload them manually.

Reply:
I've opened an issue for the release process here: https://github.com/enthought/vpsearch/issues/14. I'm not sure I want to make automating the wheel building and uploading process a priority right now, but I'll indicate that that's an option on the issue.